.github/workflows/backport-pr.yml

```
@@ -0,0 +1,63 @@
+      - master
+      - "v*"
+
+jobs:
+  check-backport:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Check if PR is a backport
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:27: Double quote to prevent globbing and word splitting

(shellcheck)